ovs/update/volumedriver/updater.py

```diff
@@ -226,26 +213,20 @@ def migrate_master_mds(cls, storagerouter, max_chain_size=100, group_timeout=10
         """
         cls.logger.info("Starting MDS migrations")
         while True:
-            hosted_vdisk_guids = storagerouter._vdisks_guids()
```

Comment:
Dont use signatures here as it will lock up. Discarded tasks are still being waited for

The reason for the catchup (see comment above) is to ensure all slaves are caught up with their master